test/helpers/index.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+  isDocData: function(response) {
+    var defined = false;
+    for (var key in response) {
```

Comment:
The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.